hub/static/js/explore.esm.js

```diff
@@ -210,11 +207,41 @@ const app = createApp({
           }).addTo(this.map)
         })
     },
+    updateMap() {
+      if (!this.map) { this.setUpMap().then( this.filterMap ) } else { this.filterMap() }
+    },
+    filterMap() {
+      fetch(this.url('/explore.json'))
+        .then(response => response.json())
+        .then(data => {
+          var features = {};
+          data.features.forEach((feature) => {
+            features[feature.properties.PCON13CD] = feature.properties;
+          });
+
+          window.geojson.eachLayer(function (layer) {
+            if ( features[layer.feature.properties.PCON13CD] ) {
+              var props = features[layer.feature.properties.PCON13CD];
+              // "show" the feature
+              layer.setStyle({
+                opacity: 1,
+                fillColor: props.color,
+                fillOpacity: props.opacity
+              })
+            } else {
+              // "hide" the feature
+              layer.setStyle({
```

Comment:
@struan Note this style gets "reset" (to 100% opacity) when you hover over "hidden" constituencies, because of the `window.geojson.resetStyle` `mouseout` listener, above.

![tracing](https://user-images.githubusercontent.com/739624/207620686-be160224-aa97-4582-a62b-5e539e3cc27d.gif)